db/seeds.rb

```diff
@@ -11,11 +9,787 @@
+UserQuery.create(id: '1004', query: 'cheap', num_count: '2516', query_type: 'DealsFeed', created_at: Date.today.beginning_of_day)
+UserQuery.create(id: '1005', query: 'salmon', num_count: '1891', query_type: 'DealsFeed', created_at: Date.today.beginning_of_day)
+UserQuery.create(id: '1006', query: 'local', num_count: '2019', query_type: 'DealsFeed', created_at: Date.today.beginning_of_day)
+UserQuery.create(id: '1007', query: 'amazing deals', num_count: '1721', query_type: 'DealsFeed', created_at: Date.today.beginning_of_day)
+UserQuery.create(id: '1008', query: 'limited time', num_count: '4912', query_type: 'DealsFeed', created_at: Date.today.beginning_of_day)
+UserQuery.create(id: '1009', query: 'savings', num_count: '3612', query_type: 'DealsFeed', created_at: Date.today.beginning_of_day)
+UserQuery.create(id: '1010', query: 'buffet', num_count: '3812', query_type: 'DealsFeed', created_at: Date.today.beginning_of_day)
+
+# Seed Data for number of wish listers
+i = 1000
+while i < 1132
+  Wish.create(id: i, venue_id: 1000, user_id: i)
+  i = i + 1
+end
+
+while i < 1301
+  Wish.create(id: i, venue_id: 1001, user_id: i)
+  i = i + 1
+end
+
+while i < 1451
+  Wish.create(id: i, venue_id: 1002, user_id: i)
+  i = i + 1
+end
+
+# Seed DealAnalytics for View count for deal 1000
+view_count = 1000
+start_date = DateTime.parse("2015-09-20 00:00:00")
+end_date = DateTime.parse("2015-10-20 00:00:00")
+while start_date <= end_date
+  limit = view_count + rand(20..40)
+  while view_count < limit
+    Viewcount.create(id: view_count, deal_id: 1000, user_id: view_count, created_at: start_date)
+    view_count = view_count + 1
+  end
+  start_date = start_date + 1
+end
+num_view_count = view_count - 1000
+unique_view_count = num_view_count- rand(50..90)
+DealAnalytic.create(id: 1000, deal_id: 1000, view_count: num_view_count, unique_view_count: unique_view_count, redemption_count: 0)
+
+# Seed DealAnalytics for View count for deal 1001
+redemption = 1000
+start_date = DateTime.parse("2015-10-01 00:00:00")
+end_date = DateTime.parse("2015-10-20 00:00:00")
+while start_date <= end_date
+  limit = redemption + rand(rand(10..30)..rand(60..100))
+  venue_id = rand(1000..1002)
+  while redemption < limit
+    Redemption.create(id: redemption, deal_id: 1001, user_id: redemption, venue_id: venue_id, created_at: start_date)
+    redemption = redemption + 1
+  end
+  start_date = start_date + 1
+end
+num_redemption = redemption - 1000
+
+starting_view_count = view_count
+start_date = DateTime.parse("2015-09-27 00:00:00")
+end_date = DateTime.parse("2015-10-20 00:00:00")
+while start_date <= end_date
+  limit = view_count + rand(rand(50..90)..rand(100..150))
+  while view_count < limit
+    Viewcount.create(id: view_count, deal_id: 1001, user_id: view_count, created_at: start_date)
+    view_count = view_count + 1
+  end
+  start_date = start_date + 1
+end
+num_view_count = view_count - starting_view_count
+unique_view_count = num_view_count- rand(50..90)
```

Comment:
Do we still need to update DealAnalytic now? Cos the method to set view count has already been implemented